faq/tidb-faq.md

```diff
@@ -41,6 +41,14 @@ Yes, it is. When all the required services are started, you can use TiDB as easi
 
 Currently, TiDB supports the majority of MySQL 5.7 syntax, but does not support trigger, stored procedures, user-defined functions, and foreign keys. For more details, see [Compatibility with MySQL](/mysql-compatibility.md).
 
+When using the MySQL 8.0 client, if it fails to connect to TiDB, try to add the `default-auth` and `default-character-set` options:
+
+```
+mysql -h 127.0.0.1 -u root -P 4000 --default-auth=mysql_native_password --default-character-set=utf8
+```
+
+This problem is caused by charset and [authentication scheme related compatibility](/security-compatibility-with-mysql.md).
```

Comment:
```suggestion
This problem occurs because MySQL 8.0 changes the [password encryption](/security-compatibility-with-mysql.md) default in MySQL 5.7. To solve this problem, you need to add the options above to specify using the old encryption method.
```

Reply:
Small nit: s/password encyption/authentication plugin/

This is the mysql terminology for it. It is useful to keep it the same because encryption is ambiguous (over the wire, at rest etc).

Reply:
@tiancaiamao PTAL